src/swift/flwr/README.md

```diff
@@ -0,0 +1,34 @@
+# flwr Swift SDK
+
+This package implements the Swift SDK for Flower - a friendly federated learning framework https://flower.dev/
+
+## Installation
+
+You can download the Flower project and integrate the package manually.
+
+## Usage
+
+A comprehensive example is available in: [```examples/ios/```](https://github.com/adap/flower/tree/main/examples/ios). To give information about the usage structurally: 
+
+```
+import flwr
+
+...
+let mlFlwrClient = MLFlwrClient(layerWrappers: layerWrappers, dataLoader: dataLoader, compiledModelUrl: compiledModelUrl)
+let flwrGRPC = FlwrGRPC(serverHost: hostname, serverPort: port)
+startFlwrGRPC(client: mlFlwrClient) {
+    // completion handler
+    print("Federated learning completed")
+}
+...
+```
```

Comment:
I feel a bit lost here. I think it would be helpful to either just refer to the example or provide a bit more overview of how the Swift Client SDK is structured.